sympy/physics/mechanics/loads.py

```diff
@@ -0,0 +1,177 @@
+from abc import ABC
+from collections import namedtuple
+from sympy.physics.mechanics.body_base import BodyBase
+from sympy.physics.vector import Vector, ReferenceFrame, Point
+
+__all__ = ['LoadBase', 'Force', 'Torque']
+
+
+class LoadBase(ABC, namedtuple('LoadBase', ['location', 'vector'])):
```

Comment:
Do we want `location` and `vector` as part of the "public" API given that we're giving both `Force` and `Torque` their own uniquely-named attributes for these? Could they be `_location` and `_vector` instead?

Reply:
Can make the private. Reason to keep them public is to let people treat `Force` and `Torque` more similar.

Reply:
Just found out another reason. `namedtuple` does not accept fields with leading underscores.